kakao/KakaoLoginAndroid/app/proguard-rules.pro

```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
+# By default, the flags in this file are appended to flags specified
+# in /Users/kevin.kang/Library/Android/sdk/tools/proguard/proguard-android.txt
```

Comment:
This file contains some references to your name "kevin.kang" and seems unused so maybe remove it?